app/jobs/update_message_object_name_job.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateMessageObjectNameJob < ApplicationJob
```

Comment:
Toto je iba tmp job, ktory potom vyhodime. Bol by problem, to vsetko poupdatovat v migracii, tak to mame vymyslene takto.